src/modules/service-catalog/components/service-catalog-item/ItemRequestForm.tsx

```diff
@@ -0,0 +1,103 @@
+import { Tagger } from "../../../new-request-form/fields/Tagger";
+import { TextArea } from "../../../new-request-form/fields/textarea/TextArea";
+import type { Field } from "../../../new-request-form/data-types";
+import { LookupField } from "../../../new-request-form/fields/LookupField";
+
+const Form = styled.form`
+  display: flex;
+  flex-direction: column;
+  gap: ${(props) => props.theme.space.md};
+`;
+
+interface ItemRequestFormProps {
+  requestFields: Field[];
+  baseLocale: string;
+  hasAtMentions: boolean;
+  userRole: string;
+  userId: number;
+  defaultOrganizationId: string | null;
+  handleChange: (field: Field, value: string | string[] | boolean) => void;
+}
+
+const ItemRequestForm = ({
+  requestFields,
+  baseLocale,
+  hasAtMentions,
+  userRole,
+  userId,
+  defaultOrganizationId,
+  handleChange,
+}: ItemRequestFormProps) => {
+  return (
+    <Form>
+      {requestFields !== undefined &&
+        requestFields.map((field) => {
+          switch (field.type) {
+            case "text":
+            case "integer":
+            case "decimal":
+            case "regexp":
+              return <Input key={field.name} field={field} />;
+            case "checkbox":
+              return (
+                <Checkbox
+                  field={field}
+                  onChange={(value: boolean) => handleChange(field, value)}
+                />
+              );
+            case "date":
+              return (
+                <DatePicker
+                  field={field}
+                  locale={baseLocale}
+                  valueFormat="date"
+                  onChange={(value) => handleChange(field, value)}
+                />
+              );
+            case "multiselect":
+              return <MultiSelect field={field} />;
+            case "textarea":
+              return (
+                <TextArea
+                  key={field.name}
+                  field={field}
+                  hasWysiwyg={false}
+                  baseLocale={baseLocale}
+                  hasAtMentions={hasAtMentions}
+                  userRole={userRole}
+                  brandId={0}
+                  onChange={(value) => handleChange(field, value)}
+                />
+              );
+            case "tagger":
+              return (
+                <Tagger
+                  key={field.name}
+                  field={field}
+                  onChange={(value) => handleChange(field, value)}
+                />
+              );
+            case "lookup":
+              return (
+                <LookupField
+                  key={field.name}
+                  field={field}
+                  userId={userId}
+                  organizationId={defaultOrganizationId}
+                  onChange={(value) => handleChange(field, value)}
+                />
+              );
+            default:
+              return <></>;
+          }
+        })}
+    </Form>
+  );
+};
+
+export default ItemRequestForm;
```

Comment:
We usually don't use `export default`, and I would just export the component directly for consistency